### ISSUE TITLE
Generate thumbs from existing thumb jobs error

### ISSUE THREAD

**Issue by theamnesic** [bnomei/kirby3-janitor#126]
Hi!

This [example](https://github.com/bnomei/kirby3-janitor?tab=readme-ov-file#examples) seems not working anymore:

```
janitor_thumbssite:
  type: janitor
  command: 'janitor:thumbs --site'
  label: Generate thumbs from existing thumb jobs (full site)
```

I've got this error: 

`array_shift(): Argument #1 ($array) must be of type array, null given`

Kirby 4.3.0
Janitor 4.2.0

Thanks!

**Comment by bnomei** [comment]
thanks @theamnesic 
i will take a look this weekend.

**Comment by bnomei** [comment]
@theamnesic just tried with v4.3.1 of janitor and i cant replicate with my default test setup

```
vendor/bin/kirby janitor:render --site
2 languages
1 pages
Rendering Pages...
[1] home
0 sec
1 images found in rendered content
0 pages failed rendering
1 pages rendered
```

```
vendor/bin/kirby janitor:thumbs --site --debug
0 existing thumbs
1 thumb job files found
Generating Thumbs...
✔ /Users/bnomei/Sites/kirby3-janitor/tests/media/pages/home/dc67e35951-1724053022/.jobs/test-128x.jpg.json
0 sec
0 jobs failed
1 thumbs created
```

**Comment by theamnesic** [comment]
I confirm that it is fixed with 4.3.1!

Thanks
